src/dev/build/cli.ts

```diff
@@ -43,6 +43,7 @@ if (showHelp) {
         --docker-cross-compile               {dim Produce arm64 and amd64 Docker images}
         --docker-contexts                    {dim Only build the Docker build contexts}
         --skip-canvas-shareable-runtime      {dim Don't build the Canvas shareable runtime}
+        --skip-cdn-assets                    {dim Don't build CDN assets}
```

Comment:
was this missed before?

Reply:
This was my assumption when I saw [this comment](https://github.com/elastic/kibana/blob/82f25ce1543cd872b85857c2a794d565526483fb/src/dev/build/build_distributables.ts#L118) 😄 , are certain flags intentionally not exposed?

Reply:
probably missed then, not intentional